src/clients/TokenClient.ts

```diff
@@ -183,7 +188,14 @@ export class TokenClient {
 
     const [balanceInfo, bondToken] = await Promise.all([
       Promise.all(Object.values(this.spokePoolClients).map((spokePoolClient) => this.fetchTokenData(spokePoolClient))),
-      this.hubPoolClient.hubPool.bondToken(),
+      // Make eth_call using nearest 100_000th block to take advantage of eth_call caching when blockTag is
+      // specified. This is unexpected to change so we are ok to rarely increase this block tag.
+      this.hubPoolClient.hubPool.bondToken({
+        blockTag: Math.min(
```

Comment:
I think this needs to be `Math.max()` to avoid always resolving to deploymentBlock.

```suggestion
        blockTag: Math.max(
```